scout/server/blueprints/login/views.py

```diff
@@ -45,64 +47,34 @@ def load_user(user_id):
 
 @login_bp.route("/login", methods=["GET", "POST"])
 @public_endpoint
-def login():
+def login() -> Response:
```

Comment:
Split this function in the following functions, now in controllers:
- handle_user_consent
- ldap_login
- google_login
- database_login
- validate_and_login_user